tech-far-hub/src/components/tfh.scss

```diff
@@ -272,6 +272,20 @@ li.usa-card {
     margin-top: 0.5rem;
   }
 }
+@include at-media-max("tablet-lg") {
+  .usa-navbar {
+    padding-top: 20px;
```

Comment:
I'm not sure if it helps or not, but sometimes I use @include u-padding-top(3) to use the spacing tokens from USWDS :)

Reply:
I go back and forth on the utility helpers